src/plugins/intel_cpu/src/node.h

```diff
@@ -645,6 +630,7 @@ class Node {
     Primitive prim;
     std::vector<DnnlDesriptor> descs;
 
+    const GraphContext::Ptr context;
```

Comment:
I believe the idea was to have a pointer on constant type, rather then have constant pointer, though the latter is not bad thing as well.

Reply:
Understood, changed, also because of constant context, I removed `ExecNetwork::setProperty` (not an override of virtual function) and `Graph::setProperty`, no one is calling them any way.